src/node.cc

```diff
@@ -187,6 +187,7 @@ static v8::Platform* default_platform;
 
 #ifdef __POSIX__
 static uv_sem_t debug_semaphore;
+static const short NUM_SIGNALS = 32;
```

Comment:
Can you name it e.g. `kNumSignals` or `kMaxSignal`?  ALL_CAPS should be reserved for macros.  LGTM apart from that.